src/Ocelot/Requester/QosDelegatingHandlerDelegate.cs

```diff
@@ -1,7 +1,8 @@
-using Ocelot.Configuration;
-using Ocelot.Logging;
+using Microsoft.AspNetCore.Http;
+using Ocelot.Configuration;
+using Ocelot.Logging;
 
 namespace Ocelot.Requester
 {
-    public delegate DelegatingHandler QosDelegatingHandlerDelegate(DownstreamRoute route, IOcelotLoggerFactory logger);
+    public delegate DelegatingHandler QosDelegatingHandlerDelegate(DownstreamRoute route, IHttpContextAccessor contextAccessor, IOcelotLoggerFactory loggerFactory);

```

Comment:
```suggestion
    public delegate DelegatingHandler QosDelegatingHandler(DownstreamRoute route, IHttpContextAccessor contextAccessor, IOcelotLoggerFactory loggerFactory);
```
So, remove "Delegate" suffix

Reply:
Ok

Reply:
Not OK! 😛 Where is the change? The proof [here](https://github.com/ThreeMammals/Ocelot/pull/1753/files#diff-586799ba26fd065bccd320cb27577637fa83b08a9ec758a28c792294115086b0R7)

Why not to press the magic button "Commit suggestion"? 😉 
The rest of work is renaming all references.
OR
You can rename manually by pressing F2 button onto the class definition. This is the best option.

Reply:
I was wondering if we could just remove it. Why this abstraction?

Reply:
Is it of any use?

Reply:
I mean, do we have other qos implementations? Why bother?

Reply:
A developer's joke... `DelegatingHandlerDelegateHandlerDelegateFactory` 😄 
I saw such classes written by Tom.